Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -197,17 +197,17 @@ extension Module  {
 
     /// Converts build settings dictionary to a Plist object.
     ///
-    /// Adds string values in dictionaries as is and array values are quoted and then converted
-    /// to a string joined by whitespace.
+    /// Adds string values in dictionaries as is and array values are converted to a
+    /// string joined by whitespace.
     private func toPlist(_ buildSettings: [String: Any]) -> Plist {
         var buildSettingsPlist = [String: Plist]()
         for (k, v) in buildSettings {
             switch v {
             case let value as String:
                 buildSettingsPlist[k] = .string(value)
             case let value as [String]:
-                let escaped = value.map { "\"" + Plist.escape(string: $0) + "\"" }.joined(separator: " ")
-                buildSettingsPlist[k] = .string(escaped)
+                let joined = value.joined(separator: " ")
```

Comment:
Ah makes sense. I see how that was working now. 

Would using a plist array for the values work? I see that Xcode will generate an array to store multiple OTHER_LINKER_FLAGS values for example. It seems more straightforward.